tests/tools/onert_train/src/args.cc

```diff
@@ -199,6 +201,72 @@ void Args::Initialize(void)
     }
   };
 
+  // supported loss list and it's default value
+  std::vector<NNFW_TRAIN_LOSS> valid_loss = {
+    NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR,
+    NNFW_TRAIN_LOSS_CATEGORICAL_CROSSENTROPY,
+  };
+  NNFW_TRAIN_LOSS default_loss = NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR;
```

Comment:
If `loss` is introduced in training and to support new loss in `tools/onert_train`, we can do it by adding one entity here.